torch_sparse/matmul.py

```diff
@@ -74,21 +74,6 @@ def spmm_max(src: SparseTensor,
 
     return torch.ops.torch_sparse.spmm_max(rowptr, col, value, other)
 
-
-def spmm(src: SparseTensor, other: torch.Tensor,
```

Comment:
I wonder if we should keep these functions for now and just utilize PyTorch `matmul` internally, i.e., `src.to_coo_sparse_tensor().matmul(other)`?